sdcm/sct_config.py

```diff
@@ -1140,8 +1133,8 @@ def __init__(self):
 
         # 4) assume multi dc by n_db_nodes set size
         if 'aws' in cluster_backend:
-            num_of_regions = len(self.get('region_name', '').split())
-            num_of_db_nodes_sets = len(str(self.get('n_db_nodes', '')).split(' '))
+            num_of_regions = len((self.get('region_name') or '').split())
+            num_of_db_nodes_sets = len(str(self.get('n_db_nodes') or '').split(' '))
```

Comment:
`n_db_nodes` is mandatory, need to fail the test. Put an assert if it is empty or 0

Reply:
it's part of required params and assert with all other requirement params

Reply:
removed "or ''"